src/components/ContactTop/index.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import { charityAPI } from '../../clients/charity';
+class ContactTop extends Component {
+  state = {
+    soicilIconsList: [],
+    loading: false,
+    error: false
+  };
+
+  componentDidMount() {
+    charityAPI('/socialmedias')
+      .then(({ data: soicilIconsList }) => {
+        this.setState({ soicilIconsList });
+      })
+      .catch(error => {});
+  }
+  render() {
+    if (this.state.loading) {
+      return <div>loading data </div>;
```

Comment:
You will never reach this code.
I'll leave it to you to know why 😉

Reply:
Because I did not set the initial state of `loading `  as  **_true_**. So that the state of `loading `will be always `false`.  🤔 

 ```
 componentDidMount() {
    charityAPI('/socialmedias')
      .then(({ data: soicilIconsList }) => {
        this.setState({ 
          soicilIconsList,
          loading:false
         });
      })
      .catch(error => {
        this.setState({
          error:true,
          loading:false
        })
      });
```